wsi_deid/web_client/views/ItemView.js

```diff
@@ -451,11 +524,52 @@ wrap(ItemView, 'render', function (render) {
         return map.layers().filter((l) => l instanceof window.geo.annotationLayer)[0];
     };
 
+    const toggleAuxImageMapDisplay = (map, image, showMap) => {
+        // Toggle showing the image or a geojs map of the image.
```

Comment:
In other Girder-3 projects we use a jsdoc style to document functions.  That is, just above the function you'd do:
```
/** Toggle ...
 *
 * @param {object} map The container of the geojs map
 * @param {object} image The container of the associated image
 * @param {boolean} showMap Truthy to show the map, falsy to show the original image.
```

Reply:
Comment has been updated to JSDoc in f73524d